CMakeLists.txt

```diff
@@ -1098,6 +1099,7 @@ if(BUILD_STUB)
             src/system/sdl/main.c
             src/studio/screens/run.c
             src/studio/screens/menu.c
+            src/studio/screens/main_menu.c
```

Comment:
Not sure here, because I'm not sure what `BUILD_STUB` does?

Reply:
It's ok, I use the `BUILD_STUB` option to build the export binaries without editors.

Reply:
Ah so that's part of the regular build process but to test it I'd want to do a Mac export say and run that individually?

Reply:
Unfortunately, it's not set up for local testing yet, you can only test it when I upload the stubs to the server.
I'll have to think about how to test stubs locally.

Reply:
Wait, so aren't the export binary part of the compiled project - aren't they needed if I do an export?

Reply:
TIC-80 downloads export binaries from the server, we have about 48 binaries for every supported OS and script and can't embed all of them to the main binary :)